frontend/app/src/features/auth/api/usePostLogout.ts

```diff
@@ -0,0 +1,34 @@
+import { backendApi } from "@/lib";
+import { useMutation, useQueryClient } from "react-query";
+import { GET_AUTH_QUERY_KEY } from "../../../hooks/useAuth";
+import { useToast } from "@chakra-ui/react";
+
+type PostLogoutRespose = {
```

Comment:
Can we expand this type to a generic API Response type (Jsend) for our APIs in/types? Since we are standardising all types to JSend Format so something like `type ApiResponse<T>` then all our API types can extend it.

Also typo here in PostLogoutRespose

Reply:
didn't do exactly what you suggested as I wanted to include zod for validation (since we have so many services and it's hard to guarantee)

Changed in [0710c93](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g03/pull/8/commits/0710c93d088f4082e5492dddf606a378c64bab92)